numpy/core/src/common/simd/estrin.h

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
I don't think it's that common, I suggest moving them close to the context inside (dispatch-able source) or in a separate header inside `umath` dir under the prefix `simd_` rather than `npyv_` in case you plan to share these macros with other sources.